pkg/client/issuecredential/client.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const (
 	// web redirect decorator.
-	webRedirectDecorator  = "~web-redirect"
+	webRedirectDecorator  = "~web_redirect"
```

Comment:
this is DIDCOmmV1 decorator

Reply:
In case of DIDCOmmV2, web redirect is in headers.

Reply:
web_redirect change reverted